polling_stations/apps/data_collection/management/commands/import_welwyn_hatfield.py

```diff
@@ -30,6 +30,14 @@ def get_station_point(self, record):
         return None
 
     def station_record_to_dict(self, record):
+        if record.code == "BFB":
```

Comment:
we're doing this same one in a slightly different way a few lines down. Just for tidyness, can we just do it once.